### PR TITLE
Add episode 4 kanban

### LINKED ISSUE #5: Episode 4: What are the Kanban basics?
Still looking for a good resource

Learning Objectives still undergoing changes:

Learning Objectives:
- Describe the characteristics of a kanban board
- Use kanban cards to track the state of work 
- Understand the importance of setting WIP limits

### PR DESCRIPTION
**Pull request by SarahAlidoost**
closes #5

### CODE REVIEW

File: _episodes/04-kanban-intro.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+- "Understand how to use a Kanban board"
+keypoints:
+- "Kanban consists of cards, columns, and continuous improvement."
+- "Main elements of continuous improvements: Commitment point, Work In Progress
+  (WIP) Limits, and Delivery point."
+---
+
+## What is Kanban?
+
+Kanban is a process managemnet method. It balances demands with available
```
Comment (raar1) [inline _episodes/04-kanban-intro.md]:
```suggestion
Kanban is a process management method. It balances demands with available
```

File: _episodes/04-kanban-intro.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+![Kanban board](../fig/kanban_board.png)
+
+### Cards
+
+Team members divide the work into some work-items and write onto cards
+(stickies, or tickets), usually one per card. For Agile teams, each card could
+represent one user story. Once on the board, these cards help everyone
+understand what the team is working on.
+
+Also, each card should have "acceptance criteria" (AC). AC is a set of
```
Comment (raar1) [inline _episodes/04-kanban-intro.md]:
```suggestion
Each card should also have "acceptance criteria" (AC). AC is a set of
```

File: _episodes/04-kanban-intro.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+> acceptance criteria (AC). However, it is a good practice if team members
+> contribute to writing AC.
+>
+{: .callout}
+
+### Columns
+
+Columns together show the process or workflow. Each column represents a specific
+activity, for example “To Do", “In Progress”, "Done". Cards flow through the
+workflow until completion, i.e. from the left to right side of the board.
```
Comment (raar1) [inline _episodes/04-kanban-intro.md]:
```suggestion
workflow until completion, i.e. from the left to the right side of the board.
```

File: _episodes/04-kanban-intro.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+
+- What if your team works on more than one project/product?
+
+If your team works on several projects/products, you can add one row (Swim Lane)
+for each of them. As an alternative, your team may decide to have a time-boxed
+workflow and work on one project at a time, see the section "Combining Scrum and
+Kanban (Scrumban)" in this lesson.
+
+> ## More examples of Kanban workflow (i.e. columns)
```
Comment (raar1) [inline _episodes/04-kanban-intro.md]:
```suggestion
> ## More examples of Kanban workflows (i.e. columns)
```

File: _episodes/04-kanban-intro.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+The delivery point is the end of a kanban workflow, i.e. all cards from “To Do"
+moved to "Done". The team’s goal is to move cards from the commitment point to
+the delivery point as fast as possible.
+
+> ## Other Kanban boards
+>
+> Have a look at the examples of Kanban workflow (mentioned above).
+>
+> - Which of those can be used for the projects in your team?
+> - What are the other workflow that you are implementing in your team?
```
Comment (raar1) [inline _episodes/04-kanban-intro.md]:
```suggestion
> - What other workflows, if any, are you implementing in your team?
```

File: _episodes/04-kanban-intro.md [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+>
+{: .discussion}
+
+> ## Designing a kanban board
+>
+> Imagin that you are working on a project.
```
Comment (raar1) [inline _episodes/04-kanban-intro.md]:
```suggestion
> Imagine that you are working on a project.
```

File: _episodes/04-kanban-intro.md [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+>
+{: .discussion}
+
+## Combining Scrum and Kanban (Scrumban)
+
+Kanban is great for processes focused on continuous delivery with changing
+priorities. On the other hand, Scrum divides work into a series of time-boxed
+iterations called sprints (see [episode 3](./03-scrum-into.md)).
+
+But maybe a combination of Scrum and Kanban (it is called "Scrumban") is what
```
Comment (raar1) [inline _episodes/04-kanban-intro.md]:
```suggestion
But maybe a combination of Scrum and Kanban (known as "Scrumban") is what
```

File: _episodes/04-kanban-intro.md [hunk 8]
```diff
@@ -0,0 +1,168 @@
...
+your team would benefit the most from.
+
+Scrumban manifests itself in different ways depending on your team
+characteristics. The team may use team meetings (like retrospective and
+planning) from Scrum and a continuous workflow with WIP limits from Kanban.
+
+> ## Discussion
+>
+> Imagin that you are working with a Kanban board. One of the work-item in “In
```
Comment (raar1) [inline _episodes/04-kanban-intro.md]:
```suggestion
> Imagine that you are working with a Kanban board. One of the work-items in “In
```

**Review by raar1 — APPROVED** [review]
This is very nice, and I think it is ready to merge (after one or two minor text changes I've suggested below). The only thing I can think of that might be missing is 'solutions' to some of the discussion questions, such as the last one perhaps (How do you deal with a blocked item in the Kanban workflow). If you agree then we could make an issue for it e.g. "Add example solutions to discussion questions in episode3". But this isn't super important so let's just merge this PR soon. Good work 👍

**Comment by SarahAlidoost** [comment]
> This is very nice, and I think it is ready to merge (after one or two minor text changes I've suggested below). The only thing I can think of that might be missing is 'solutions' to some of the discussion questions, such as the last one perhaps (How do you deal with a blocked item in the Kanban workflow). If you agree then we could make an issue for it e.g. "Add example solutions to discussion questions in episode3". But this isn't super important so let's just merge this PR soon. Good work +1

Thanks for reviewing. I agree and made an issue #40  for that.

**Pull request merged**